test/e2e/framework/util.go

```diff
@@ -3454,8 +3454,8 @@ func IssueSSHCommand(cmd, provider string, node *v1.Node) error {
 	return nil
 }
 
-// NewHostExecPodSpec returns the pod spec of hostexec pod
-func NewHostExecPodSpec(ns, name string) *v1.Pod {
+// NewExecPodSpec returns the pod spec of hostexec pod
```

Comment:
s/hostexec/exec

Reply:
Hm, debatable. I think "hostexec pod" refers to the fact that the pod is using the hostexec image, which is still true.

Reply:
why did the function name change?